Xamarin.Essentials/Browser/Browser.shared.cs

```diff
@@ -1,28 +1,47 @@
 ﻿using System;
+using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 
+#if NETSTANDARD1_0
+public struct Color
+{
+    public int Value { get; set; }
+
+    public byte A => (byte)(Value >> 24);
+
+    public byte R => (byte)(Value >> 16);
+
+    public byte G => (byte)(Value >> 8);
+
+    public byte B => (byte)Value;
+}
+#else
+using System.Drawing;
+[assembly: TypeForwardedTo(typeof(Color))]
```

Comment:
Rather use the full name instead of the using: `[assembly: TypeForwardedTo(typeof(System.Drawing;.Color))]`

I think there are rules about where usings can go...

Reply:
Stylecop did not complain. If I remove the using the android build fails, not sure why.